content/applications/finance/accounting/receivables/customer_invoices/snailmail.rst

```diff
@@ -4,42 +4,51 @@
 Send your invoices by post (Snailmail)
 ======================================
 
-Direct mail is a great way to capture individuals’ attention at a time where inboxes are always
-full. Odoo allows you to send invoices and follow-up reports by post, worldwide, directly from
-your database.
+Direct mail is a great way to capture individuals’ attention when inboxes are always full. Odoo
+allows you to send invoices and follow-up reports by post worldwide, directly from your database.
 
-Set up Snailmail
-================
+Snailmail configuration
+=======================
 
-| Go to :menuselection:`Accounting --> Settings --> Snailmail` and activate the feature.
-| If you would like this option to be a default feature, enable *Send by Post* under
-  *Default Sending Options*.
+Go to :menuselection:`Accounting --> Configuration --> Settings --> Customer invoices section` to
+activate Snailmail.
+
+To make it a by-default feature, select :guilabel:`Send by Post` in the
+:guilabel:`Default Sending Options` section.
 
 .. image:: snailmail/setup_snailmail.png
    :align: center
    :alt: Under settings enable the snailmail feature in Odoo Accounting
 
-Send your invoices by post
---------------------------
-
-Open your Invoice, click on the *Send & Print* button and select *Send by Post*.
+Send your invoice by post
+-------------------------
 
-.. image:: snailmail/invoice_bypost.png
-   :align: center
-   :alt: Overview of an invoice and the option send by post available in Odoo Accounting
+Open your invoice, click on :guilabel:`Send & Print` and select :guilabel:`Send by Post`. Make sure
+your customer’s address is set correctly, including the country, before sending the letter.
 
 .. important::
-   Make sure to have your customer’s address set correctly, and that it includes a country,
-   before sending the letter.
+   Your document must respect the following rules to pass the validation before being sent:
+
+   - Margins must be 5 millimeters on all sides. As Odoo forces the outer margins by filling them
+     with white before sending the snailmail, it can results in the user's custom being cut off if
+     it protrudes into the margins. To check the margins, go to :menuselection:`General Settings
+     --> Technical --> Reporting section: Paper format`.
```

Comment:
I couldn't find "technical" and the "reporting section"; here's the path I took to find the paper format in v14 
```suggestion
     it protrudes into the margins. To check the margins, go to :menuselection:`Settings --> General
     Settings`, scroll down to the :guilabel:`Business Documents` section, and open the external
     link.
```

Also, I couldn't find it in Odoo 15 and Odoo 16 --> do you know if there's a way to check the margins in these versions? If not, it should be removed for the two versions

Reply:
Actually, "Technical" is available in Debug mode, but I understood we avoid to advise users to activate it. I updated as per your suggestion. For V15 and V16, I will investigate to see what are the options.